RELEASE_NOTES.md

```diff
@@ -39,14 +39,14 @@ the following platforms:
 
 To run _Kotlin/Native_ JDK 8 or Java 9 (JDK) for the host platform has to be installed.
 
-On macOS it also requires Xcode 9.4.1 or newer to be installed.
+On macOS it also requires Xcode 10.0 or newer to be installed.
```

Comment:
Not sure if this is correct, we likely can work (not compile compiler) with older Xcode.